BlockHeaderDone

Kyzenko opened the pull request:
-[DEMO LINK](https://Kyzenko.github.io/layout_miami/)

SanyaBratashchuk reviewed (changes requested):
1. form should be reset after submit
![image](https://github.com/mate-academy/layout_miami/assets/97547824/2851ce94-06f5-441b-b5fb-bb4a67526941)
2. Add hover effects for these links 
![image](https://github.com/mate-academy/layout_miami/assets/97547824/b21dd454-44b7-4926-966e-f3433b056c27)

SanyaBratashchuk reviewed (changes requested):
comments ⏫

SanyaBratashchuk reviewed (changes requested):
[IGNORE IT] Test reject

SanyaBratashchuk reviewed (changes requested):
Fix previous comments and don't create new bugs🤣 (404 after click on logo )
![image](https://github.com/mate-academy/layout_miami/assets/97547824/1f9a6d0f-4fa8-4cbf-bed8-5967c1d7b39d)

Kyzenko commented:
> Fix previous comments and don't create new bugs🤣 (404 after click on logo ) ![image](https://user-images.githubusercontent.com/97547824/239751121-1f9a6d0f-4fa8-4cbf-bed8-5967c1d7b39d.png)

Ahahah🤣 I deleted href="/" and added href="#home" then deleted href="#home" and added href="/". 
I didn't even notice it at first and did not understand what the matter was.

DarkMistyRoom reviewed (changes requested):
Great job! Just a few moments:
- The font in the texarea is different
<img width="482" alt="image" src="https://github.com/mate-academy/layout_miami/assets/98764641/dfcfc3e2-d18c-47eb-bafb-c06adfa463c9">

- The form submitting shouldn't reload the page, just reset the form
- Please check the side indents on the tab. They look too big
<img width="482" alt="image" src="https://github.com/mate-academy/layout_miami/assets/98764641/d60351e1-711d-48b5-9bb6-ea5f28423e9f">

- check the background on the mobile. It looks different in the mockup
<img width="482" alt="image" src="https://github.com/mate-academy/layout_miami/assets/98764641/bd671acf-b275-4c19-9456-2e2be2374270">

Kyzenko commented:
> Great job! Just a few moments:
> 
> * The font in the texarea is different
> 
> <img alt="image" width="482" src="https://user-images.githubusercontent.com/98764641/241150947-dfcfc3e2-d18c-47eb-bafb-c06adfa463c9.png">
> * The form submitting shouldn't reload the page, just reset the form
> * Please check the side indents on the tab. They look too big
> 
> <img alt="image" width="482" src="https://user-images.githubusercontent.com/98764641/241151567-d60351e1-711d-48b5-9bb6-ea5f28423e9f.png">
> * check the background on the mobile. It looks different in the mockup
> 
> <img alt="image" width="482" src="https://user-images.githubusercontent.com/98764641/241151723-bd671acf-b275-4c19-9456-2e2be2374270.png">

Kyzenko commented:
> * Please check the side indents on the tab. They look too big

> <img alt="image" width="482" src="https://user-images.githubusercontent.com/98764641/241151567-d60351e1-711d-48b5-9bb6-ea5f28423e9f.png">
> * check the background on the mobile. It looks different in the mockup

> <img alt="image" width="482" src="https://user-images.githubusercontent.com/98764641/241151723-bd671acf-b275-4c19-9456-2e2be2374270.png">


I don't quite understand what the error is.

loralevitska reviewed (changes requested):
Good job! There are things that should be fixed:

- use svg for logo for better quality 
- remove redundant elements from photo (just download another photo without this element) Fix all photos 
![image](https://github.com/mate-academy/layout_miami/assets/70009205/1de964d9-8312-46fa-a6cc-4fdb88829f19)
- Remove autocomplete background 
![image](https://github.com/mate-academy/layout_miami/assets/70009205/0cfabae6-cd5a-4a73-83c7-5d98e702b1fe)
- don't reload page after form submit, just reset the form
